test/docfx.Test/lib/GitUtilityTest.cs

```diff
@@ -48,8 +48,8 @@ public static async Task GetCommitsSameAsGitExe(string file)
                     string.Join("\n", lib.Select(c => $"{c.Sha}|{c.Time.ToString("s")}{c.Time.ToString("zzz")}|{c.AuthorName}|{c.AuthorEmail}")));
 
                 // another branch
-                exe = Exec("git", $"--no-pager log --format=\"%H|%cI|%an|%ae\" origin/test -- \"{pathToRepo}\"", repo);
-                lib = commitsProvider.GetCommitHistory(pathToRepo, "test");
+                exe = Exec("git", $"--no-pager log --format=\"%H|%cI|%an|%ae\" a050eaf -- \"{pathToRepo}\"", repo);
```

Comment:
>a050eaf [](start = 79, length = 7)

also add a normal other branch name test?

Reply:
this test fails if your local branch does not have the same commit as server, or if origin is a forked repo that does not have a test branch (like me), a commit hash will always have a the same result.